sdk/identity/Azure.Identity/tests/CredentialTestBase.cs

```diff
@@ -109,6 +109,55 @@ public async Task RespectsIsSupportLoggingEnabled([Values(true, false)] bool isS
             Assert.True(_listener.EventData.Any(d => d.Payload.Any(p => p.ToString().StartsWith($"{expectedPrefix} MSAL"))));
         }
 
+        [Test]
+        [TestCase(EventLevel.Informational)]
```

Comment:
Should we add `EventLevel.Critical`, as the newly added level?

Reply:
I excluded them because in this test we can't reliably produce critical MSAL events. This subset is enough to validate that the levels are dynamic.